src/main/java/com/refinedmods/refinedstorage/apiimpl/autocrafting/CraftingManager.java

```diff
@@ -94,32 +91,30 @@ public void cancel(@Nullable UUID id) {
     }
 
     @Override
-    @Nullable
-    public ICraftingTask create(ItemStack stack, int quantity) {
```

Comment:
create methods can be easily combined

Reply:
I don't see how. One is for ItemStack, one is for FluidStack? Also the getPattern calls are specific to the stack type.

Reply:
Oh. Nvm. You can combine most of the code into an extra method if you want to. 

Fully combining it requires making `CraftingManager#getPattern` and `CraftingRequestInfo` generic + a generic subclass.

Reply:
I'll leave it as-is